public/content/translatathon/details/index.md

```diff
@@ -0,0 +1,78 @@
+
+**3) Open an untranslated file**
+- Find the first untranslated file to start translating. The folders containing the source files are based on priority - (1) Homepage, 2) Essential learning, 3) Essentials, 4) Exploring, etc. so you should start translating the first folder that contains untranslated files
+- Each file has a progress indicator showing how much of the translatable content in the file has been translated and approved… if translation progress for any file is below 100%, please translate it
+
+**4) Translate the untranslated strings**
+- When you open a file to translate, make sure you are only translating untranslated strings!
+- Each string has a status indicator that shows whether it’s *Translated*, *Untranslated*, or *Approved*. If a source string already has a suggested translation in your language, there is no need to translate it
+- You can also filter strings in the editor to show *Untranslated first* or *Untranslated only*
+
+For a detailed guide to navigating and using the Crowdin online editor, we recommend all Translatathon participants to read our ‘[How to translate](https://ethereum.org/en/contributing/translation-program/how-to-translate/)’ guide.
+
+To learn more about the conventions and best practices for translating ethereum.org content, yo can also check out our [translation style guide](https://ethereum.org/en/contributing/translation-program/translators-guide/).
+
+### Prizes
+
+The total prize pool for the Translatathon is 30,000$.
+
+A detailed breakdown of prizes will be announced at the end of the application period.
+
+### Evaluation process
+
+We work with [Acolad](https://www.acolad.com/), a leading localization agency, on all review and QA processes for ethereum.org content.
+
+As part of the evaluation process for the Translatathon, all translations will be subject to a QA and feedback step, where professional linguists will evaluate submissions by individual translators based on quality and accuracy.
+
+We will also be running anti-machine translation measures, with Crowdin providing some tools that automatically detect machine translations.
+
+While translation quality will not play a critical role in the scoring, any participants found using machine translation or suggesting low-quality and inaccurate translations will be disqualified and not eligible to compete for prizes!
+
+The evaluation period will take place between August 19th-28th and the results will be announced on the ethereum.org community call on August 29th.
+
+All translations will also be subject to a thorough review before being added to the website.
+
+### FAQ - Frequently asked questions
+
+<ExpandableCard title="Who can I contact if I need help or have questions, feedback, or ideas?">
+  <ul>
+    <li>In Crowdin, you can send a direct message to <a href="https://crowdin.com/profile/ethdotorg"><strong>Ethereum.org Team</strong></a></li>
```

Comment:
@corwintines  seems broken, link doesnt work in preview

Reply:
Updated this link now